sdk/storage/Azure.Storage.Blobs/api/Azure.Storage.Blobs.netstandard2.0.cs

```diff
@@ -713,6 +735,11 @@ public partial class BlobLeaseRequestConditions : Azure.RequestConditions
         public BlobLeaseRequestConditions() { }
         public string TagConditions { get { throw null; } set { } }
     }
+    public partial class BlobLegalHoldInfo
+    {
+        public BlobLegalHoldInfo() { }
+        public bool LegalHoldEnabled { get { throw null; } }
```

Comment:
Should this be `HasLegalHold` like elsewhere ?

Reply:
Changed to `HasLegalHold`